spec/index.bs

```diff
@@ -1441,6 +1438,8 @@ GPUDevice includes GPUObjectBase;
         The [=device=] that this {{GPUDevice}} refers to.
 </dl>
 
+Issue: inherit from `GPUObjectBase`?
```

Comment:
Is this related?

Reply:
used to be related when I was adding the "current agent cluster" field to the object, but now it's not needed, removed